Data/SkillStatMap.lua

```diff
@@ -975,6 +975,7 @@ return {
 },
 ["projectile_damage_+%_per_remaining_chain"] = {
 	mod("Damage", "INC", nil, ModFlag.Projectile, 0, { type = "PerStat", stat = "ChainRemaining" })
```

Comment:
I was going to say this breaks the "Follow-Through" cluster jewel notable, but apparently they updated the wording to also say "With Hits and Ailments" on that, too.

Just add a comma here and it looks good:
```suggestion
	mod("Damage", "INC", nil, ModFlag.Projectile, 0, { type = "PerStat", stat = "ChainRemaining" }),
```